dbt/models/iasworld/schema/iasworld.dweldat.yml

```diff
@@ -437,11 +518,21 @@ sources:
                   name: iasworld_dweldat_mktrsn_eq_5_mktadj_not_null
                   expression: |
                     != '5' OR mktadj IS NOT NULL
+                  select_columns:
+                    - taxyr
+                    - parid
+                    - card
+                    - who
+                    - wen
+                    - mktadj
                   config:
                     where: |
                       CAST(taxyr AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
                       AND cur = 'Y'
                       AND deactivat IS NULL
+                  meta:
+                    category: incorrect_values
+                    description: mktrsn should only be 5 if mktadj is not null
```

Comment:
This test has also started failing recently, hence we add a category, description, and additional select columns.